### PR TITLE
(blog) Week 16 blog post update

### PR DESCRIPTION
**Pull request by Disha-maker**
This is the new PR, everything looks good

### CODE REVIEW

File: _lifeofinternatpuppet/post_16.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Hello Everyone!
+
+Not too long to Christmas now, of course?
+
+This has been the busiest week of my placement so far, before the team wraps up for Christmas holidays!
+
+Some of the key highlights included:-
+
+**Module Release Prep**
+- [puppetlabs-motd (IAC-1308)](https://tickets.puppetlabs.com/browse/IAC-1308) (v4.3.0)
```
Comment (pmcmaw) [inline _lifeofinternatpuppet/post_16.md:21]: tagmail and tomcat are listed twice, one with pr and one with the release ticket, the duplication is unnecessary

File: _lifeofinternatpuppet/post_16.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- [puppetlabs-tomcat (IAC-1330)](https://tickets.puppetlabs.com/browse/IAC-1330)
+- [puppetlabs-tomcat (PR #426)](https://github.com/puppetlabs/puppetlabs-tomcat/pull/426) (v4.3.0)
+- [puppetlabs-tagmail (IAC-1349)](https://tickets.puppetlabs.com/browse/IAC-1349)
+- [puppetlabs-tagmail (PR #196)](https://github.com/puppetlabs/puppetlabs-tagmail/pull/196)
+
+**Module Releases**
+-[puppetlabs/accounts](https://forge.puppet.com/puppetlabs/accounts) (v6.4.0)
```
Comment (pmcmaw) [inline _lifeofinternatpuppet/post_16.md:28]: puppetlabs/accounts isnt consistent with the other module names with '-'

Comment (Disha-maker) [reply _lifeofinternatpuppet/post_16.md:26]: That's how @sheenaajay told me to format it

Comment (sheenaajay) [reply _lifeofinternatpuppet/post_16.md:26]: @Disha-maker we can make it consistent with other releases you mentioned below.

File: _lifeofinternatpuppet/post_16.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+Some of the key highlights included:-
+
+**Module Release Prep**
+- [puppetlabs-motd (IAC-1308)](https://tickets.puppetlabs.com/browse/IAC-1308)
+- [puppetlabs-tomcat (IAC-1330)](https://tickets.puppetlabs.com/browse/IAC-1330)
+- [puppetlabs-tagmail (IAC-1349)](https://tickets.puppetlabs.com/browse/IAC-1349)
+
+**Module Releases**
+-[puppetlabs/accounts](https://forge.puppet.com/puppetlabs/accounts) (v6.4.0)
```
Comment (sheenaajay) [inline _lifeofinternatpuppet/post_16.md:26]:
```suggestion
-[puppetlabs-accounts](https://forge.puppet.com/puppetlabs/accounts) (v6.4.0)
```

File: _lifeofinternatpuppet/post_16.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+**Module Release Prep**
+- [puppetlabs-motd (IAC-1308)](https://tickets.puppetlabs.com/browse/IAC-1308)
+- [puppetlabs-tomcat (IAC-1330)](https://tickets.puppetlabs.com/browse/IAC-1330)
+- [puppetlabs-tagmail (IAC-1349)](https://tickets.puppetlabs.com/browse/IAC-1349)
+
+**Module Releases**
+-[puppetlabs-accounts](https://forge.puppet.com/puppetlabs/accounts) (v6.4.0)
+-[puppetlabs-concat](https://forge.puppet.com/modules/puppetlabs/concat) (v6.4.0)
+-[puppetlabs-service](https://forge.puppet.com/modules/puppetlabs/service) (v1.4.0)
+-[puppetlabs-docker](https://forge.puppet.com/modules/puppetlabs/docker) (v3.13.0)
```
Comment (sanfrancrisko) [inline _lifeofinternatpuppet/post_16.md:29]: You need a space between the `-` and the next character.
```suggestion
- [puppetlabs-accounts](https://forge.puppet.com/puppetlabs/accounts) (v6.4.0)
- [puppetlabs-concat](https://forge.puppet.com/modules/puppetlabs/concat) (v6.4.0)
- [puppetlabs-service](https://forge.puppet.com/modules/puppetlabs/service) (v1.4.0)
- [puppetlabs-docker](https://forge.puppet.com/modules/puppetlabs/docker) (v3.13.0)
```

**Review by sanfrancrisko — CHANGES_REQUESTED** [review]

**Pull request merged**